.changelog/23510.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
can we add an entry for the `aws_s3_bucket_policy` resource as well since it's been updated as well?

Reply:
thanks for the updates @erikpaasonen ! apologies for confusion here, the `go-changelog` tools requires separate blocks so instead of adding the `resource/aws_s3_bucket_policy` @ L3, it should exist in it's own release note as 

```
```release-note:bug
resource/aws_s3_bucket_policy: Let resource be removed from tfstate if bucket deleted outside Terraform
     <-- closing backticks here too
```